sdk/websocket.go

```diff
@@ -200,8 +204,12 @@ func (c *ClientWebsocket) subsChannel(s *subscribe) error {
 			}
 
 			subName, _ := restParser(resp)
+			b := subscribeInfo{
+				name:    subName,
+				account: s.getAdd(),
+			}
 
-			e = s.buildType(subName, resp)
+			e = b.buildType(resp)
```

Comment:
The func working in goroutine. Do you sure that finish & return 'e' which is defined by this string?

Reply:
solved in: https://github.com/proximax-storage/nem2-sdk-go/pull/115/commits/aa550fdc2f6d1a028381e55fef54bcdb79b2a107